### ISSUE TITLE
Missing `packaging` module

### ISSUE THREAD

**Issue by roberto-arista** [typemytype/drawbot#503]
hey all!

I freshly installed drawbot from the master branch, and I try to import it into a script with:
```python
import drawBot
```

I get this traceback:
```
Traceback (most recent call last):
  File "/Users/robertoarista/Desktop/drawbot bug/hello.py", line 1, in <module>
    import drawBot
  File "/Users/robertoarista/Desktop/drawbot bug/.venv/lib/python3.11/site-packages/drawBot/__init__.py", line 1, in <module>
    from .drawBotDrawingTools import _drawBotDrawingTool
  File "/Users/robertoarista/Desktop/drawbot bug/.venv/lib/python3.11/site-packages/drawBot/drawBotDrawingTools.py", line 11, in <module>
    from .context import getContextForFileExt, getContextOptions, getFileExtensions, getContextOptionsDocs
  File "/Users/robertoarista/Desktop/drawbot bug/.venv/lib/python3.11/site-packages/drawBot/context/__init__.py", line 1, in <module>
    from .pdfContext import PDFContext
  File "/Users/robertoarista/Desktop/drawbot bug/.venv/lib/python3.11/site-packages/drawBot/context/pdfContext.py", line 5, in <module>
    from packaging.version import Version
ModuleNotFoundError: No module named 'packaging'
```

instead if I try to `import packaging` from the 3.129 app, no traceback.
I am running this in py3.11, will soon test it with 3.9.13.

**Comment by roberto-arista** [comment]
same outcome with 3.9.13

**Comment by justvanrossum** [comment]
You mean installed as a module? It seems `packaging` is not listed in the `setup.py` requirements. PR welcome.

**Comment by roberto-arista** [comment]
with `pip install packaging` the traceback disappears!

**Comment by roberto-arista** [comment]
fixed with #504
